tests/search/setup_search_tests.py

```diff
@@ -75,7 +78,10 @@ def _populate_db_families(db: Session, max_docs: int = VESPA_FIXTURE_COUNT) -> N
         if doc["fields"]["family_document_ref"] not in seen_family_ids:
             _create_family(db, family)
             _create_family_event(db, family)
-            _create_family_metadata(db, family)
+            if not deterministic_metadata:
```

Comment:
Do tests fail if we just make all of them deterministic?

Reply:
Yes there was like ~9 failures so given the time I just went with this approach!

Reply:
Sounds reasonable 👍